src/clients/bridges/LineaAdapter.ts

```diff
@@ -266,15 +266,17 @@ export class LineaAdapter extends BaseAdapter {
     l1Token: string,
     transferEvents: Event[]
   ): void {
+    const l2Token = this.resolveL2TokenAddress(l1Token, false); // There's no native USDC on Linea
     transferEvents.forEach((event) => {
       const txHash = event.transactionHash;
       // @dev WETH events have a _value field, while ERC20 events have an amount field.
       const amount = event.args._value ?? event.args.amount;
       outstandingTransfers[monitoredAddress] ??= {};
-      outstandingTransfers[monitoredAddress][l1Token] ??= { totalAmount: bnZero, depositTxHashes: [] };
-      outstandingTransfers[monitoredAddress][l1Token] = {
-        totalAmount: outstandingTransfers[monitoredAddress][l1Token].totalAmount.add(amount),
-        depositTxHashes: [...outstandingTransfers[monitoredAddress][l1Token].depositTxHashes, txHash],
+      outstandingTransfers[monitoredAddress][l1Token] ??= {};
+      outstandingTransfers[monitoredAddress][l1Token][l2Token] ??= { totalAmount: bnZero, depositTxHashes: [] };
+      outstandingTransfers[monitoredAddress][l1Token][l2Token] = {
+        totalAmount: outstandingTransfers[monitoredAddress][l1Token][l2Token].totalAmount.add(amount),
+        depositTxHashes: [...outstandingTransfers[monitoredAddress][l1Token][l2Token].depositTxHashes, txHash],
```

Comment:
There's probably also an opportunity to condense this with the same trick as in https://github.com/across-protocol/relayer-v3/pull/1475/files#r1593639666:

```suggestion
      const transfers = outstandingTransfers[monitoredAddress][l1Token] ??= {};
      transfers[l2Token] ??= { totalAmount: bnZero, depositTxHashes: [] };
      transfers[l2Token] = {
        totalAmount: transfers[l2Token].totalAmount.add(amount),
        depositTxHashes: [...transfers[l2Token].depositTxHashes, txHash],
```

Reply:
I'm not a huge fan of using the return type of the assignment operator - happy to accept the suggestion if there's pushback